mlir/test/Dialect/Linalg/unfold_projected_permutation.mlir

```diff
@@ -0,0 +1,71 @@
+// RUN: mlir-opt %s -split-input-file --linalg-specialize-generic-ops | FileCheck %s
+
+#projection = affine_map<(d0, d1, d2, d3, d4) -> (d2, d3, d1)>
+#identity   = affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d3, d4)>
+
+func.func @test_mixed(%x : tensor<7x8x9xf32>, %y:  tensor<5x9x7x8x10xf32>, %z :  tensor<5x9x7x8x10xf32>) ->  tensor<5x9x7x8x10xf32> {
```

Comment:
[nit] Skip "test" in ... test function names. Instead, try to convey some unique information. So, what is actually being tested here? Why "mixed"? As in, what's being mixed?

Reply:
done